client/main.js

```diff
@@ -34,10 +35,10 @@ window.addEventListener( 'beforeunload', ( event ) => {
 const rootEl = document.getElementById( 'wc-connect-admin-container' );
 
 let render = () => {
-	const WCCShippingSettings = require( 'views/shipping' );
+	const RootView = require( 'views/root/' + rootView );
```

Comment:
It can't be **that** smart. If you require `"something/" + arbitraryString`, the bundle must have all the possible bundles ready.